ddpui/ddpairbyte/airbytehelpers.py

```diff
@@ -775,7 +795,9 @@ def get_sync_job_history_for_connection(
         return None, "connection not found"
 
     res = {"history": [], "totalSyncs": 0}
-    result = airbyte_service.get_jobs_for_connection(connection_id, limit, offset)
+    result = airbyte_service.get_jobs_for_connection(
+        connection_id, limit, offset, job_types=["sync", "reset_connection"]
```

Comment:
why don't we ask for `clear` over here?

Reply:
Ahh yes

The clear jobs it seems come under `reset_connection`. Although the api documentation says `clear` is a configType but it throws an unknown error when i use it. 

However the above configuration, gives the correct results. But i have no way making out whether its a clear or reset. I dont know how airbyte shows it in its job history. (even airbyte queries on their UI with those configTypes)